core/templates/dev/head/pages/state_editor/StateResponsesDirective.js

```diff
@@ -121,7 +121,8 @@ oppia.directive('stateResponses', [
                     ruleInputs.forEach(function(ruleInput) {
                       var choiceIndex = answerChoiceToIndex[ruleInput];
                       if (rule.type === 'Equals' ||
-                          rule.type === 'ContainsAtLeastOneOf') {
+                          rule.type === 'ContainsAtLeastOneOf' ||
+                          rule.type === 'IsProperSubsetOf') {
```

Comment:
See the note that I left in ItemSelectionInputValidationService.js

Reply:
This comment is not resolved yet.

Reply:
Done, removed it.